fmn/delivery/service.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of the FMN project.
+# Copyright (C) 2017 Red Hat, Inc.
```

Comment:
You probably don't need the utf-8 header unless you want to use the sweet sweet © character here.